<launch>
	<node name="gps_reader" pkg="project" type="gps_reader.py" output="screen" />
</launch>
